airflow/contrib/operators/gcp_spanner_operator.py

```diff
@@ -197,3 +197,192 @@ def execute(self, context):
     def sanitize_queries(queries):
         if len(queries) and queries[-1] == '':
             del queries[-1]
+
+
+class CloudSpannerInstanceDatabaseDeployOperator(BaseOperator):
+    """
+    Creates a new Cloud Spanner database or if database exists it does nothing.
```

Comment:
I'd slightly rephrase:
`Creates a new Cloud Spanner database if it doesn't exist. Otherwise, does nothing.`

Reply:
I rephrased it accordingly to Deb's reviews for other operators.